src/lib/libfs.js

```diff
@@ -1935,4 +1933,17 @@ FS.staticInit();
 
   $FS_createLazyFile__deps: ['$FS'],
   $FS_createLazyFile: 'FS.createLazyFile',
-});
+
+};
+
+// Add library aliases for all the FS.<symbol> as FS_<symbol>.
```

Comment:
Alternatively, we could make a static list of these like FS_createLazyFile above. There are currently ~121 properties on the FS object though.

Reply:
I think we can probably come up with a shortlist.  I think just the ones that wamfs.js implements should be enough.  

I'd be OK with landing this version first if you prefer.

Reply:
Seems like we should delete the ones above that do the same thing (if we do land this version)